style.css

```diff
@@ -369,6 +369,185 @@ button:active {
   background: var(--colorButtonPressed);
 }
 
+/* ----- POPUP CREATION ----- */
+
+.popup__window {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--backgroundColor);
+  background-blend-mode: multiply;
+  z-index: 100;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.popup_content {
+  padding: 1.6rem;
+  width: 90%;
+  max-width: 920px;
+  max-height: 1083px;
+  margin: 5% auto 2% auto;
+  background: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 8px;
+  z-index: 200;
+}
+
+.popup-content__wrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+    "img"
+    "title"
+    "tags"
+    "description"
+    "buttons";
+  justify-content: center;
+  align-items: center;
+}
+
+#popup__close__button {
+  grid-area: 1/1/2/2;
+  align-self: start;
+  justify-self: end;
+  margin: 1rem;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  position: relative;
+}
+
```

Comment:
- [ ] Good job so far 👍👍 However, as suggested by the previous reviewer [here](https://github.com/mateoburvi/Portfolio/pull/12#discussion_r1249221892), kindly make sure to place the close button in accordance with the given Figma template✨⛏ to get a design close to the Figma template as much as possible ✌


Your design | Figma design 
---| ---|
![image](https://github.com/mateoburvi/Portfolio/assets/106119105/375c79c6-3a6d-47d6-9190-05a52d014196) | ![image](https://github.com/mateoburvi/Portfolio/assets/106119105/eb840150-5d3a-46e2-8806-cac61390edc3)

Reply:
can i have your slack? i dont have much time. please just help me approve this block and I will correct the mistakes later